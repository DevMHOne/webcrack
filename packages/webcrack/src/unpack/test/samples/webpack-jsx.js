(window['webpackJsonp'] = window['webpackJsonp'] || []).push([
  [1],
  [function (module, exports, __webpack_require__) {
    __webpack_require__.r(exports);
    var r = __webpack_require__(35250);
    function f() {
      return (0, r.jsx)("p", {});
    }
  }],
]);
