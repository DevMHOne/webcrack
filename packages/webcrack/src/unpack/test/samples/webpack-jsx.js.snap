WebpackBundle {
  "chunks": [
    WebpackChunk {
      "chunkIds": [
        "1",
      ],
      "entryIds": [],
      "modules": Map {
        "0" => WebpackModule {
          "ast": import { jsx } from "35250";
function f() {
  return <p />;
},
          "dependencies": Set {
            "35250",
          },
          "exports": Set {},
          "externalModule": undefined,
          "id": "0",
          "isEntry": false,
          "path": "0.js",
        },
      },
    },
  ],
  "entryId": "",
  "modules": Map {
    "0" => WebpackModule {
      "ast": import { jsx } from "35250";
function f() {
  return <p />;
},
      "dependencies": Set {
        "35250",
      },
      "exports": Set {},
      "externalModule": undefined,
      "id": "0",
      "isEntry": false,
      "path": "0.js",
    },
  },
  "type": "webpack",
}